src/bin/lttng/commands/enable_events.c

```diff
@@ -290,6 +290,63 @@ static int walk_command_search_path(const char *binary, char *binary_full_path)
 	return ret;
 }
 
+/*
+ * Check if the symbol field passed by the user is in fact an address or an
+ * offset from a symbol. Those two instrumentation types are not supported yet.
+ * It's expected to be a common mistake because of the existing --probe option
+ * that does support these formats.
+ *
+ * Here are examples of these unsupported formats for the --userspace-probe
+ * option:
+ * elf:/path/to/binary:0x400430
+ * elf:/path/to/binary:4194364
+ * elf:/path/to/binary:my_symbol+0x323
+ * elf:/path/to/binary:my_symbol+43
+ */
+static int warn_userspace_probe_syntax(const char *symbol)
+{
+	unsigned long addr = 0;
+	size_t offset = 0;
+	int ret = -1;
+
+	/* Check if the symbol field is an hex address. */
+	ret = sscanf(symbol, "0x%lx", &addr);
+	if (ret > 0) {
+		/* If there is a match, print a warning and return an error. */
+		ERR("Userspace probe on address not supported.");
```

Comment:
Maybe add the 'yet'? At least if you plan to add this support soon ;-)

Reply:
That fine too. It's just a teaser to let users know it will eventually come if that's the plan. So at a later release, they'll remember and try it again instead of just switching to another tracer that does it. Marketing in the code! :p